### DISCUSSION TITLE
mark extrema value in chart

### DISCUSSION THREAD

**Discussion by PatricDahl** [kalkih/mini-graph-card#705] [Ideas]
Hi,

Is it possible to have highest value marked in chart with help of extrema?
Or if to have it print on label somewhere?

BR
Patric

**Comment by ildar170975** [comment]
> to have it print on label somewhere?

You can already print extrema values:
![image](https://user-images.githubusercontent.com/71872483/150782735-0e51ec71-965f-4779-ae5d-203534885990.png)
Use `extrema: true` inside the `show:` section.
